_data/health.yml

```diff
@@ -13,9 +14,10 @@ websites:
       facebook: carilionclinic
       email_address: direct@carilionclinic.org
 
-    - name: CVS/Caremark
-      url: https://www.caremark.com
-      twitter: cvshealth
+    - name: CVS Health
```

Comment:
These are two reasonably separate entities. just switch back to caremark and remove the twitter?